### ISSUE TITLE
The problem of setting up the comparative experiment

### ISSUE THREAD

**Issue by cooker-coder** [stitsyuk/xPatch#3]
Dear Author：
Hello.

Firstly, the unified hyperparameter comparison experiment in the article is based on the experimental setting of unified hyperparameters of TimesNet. After reading the original text of TimesNet, I learned that its unified hyperparameter experimental Settings are: alignment review window, batch size, and training rounds. However, in the script file of the Github repository you published, the batch size and training rounds in `xPatch_unified.sh` are different from the Settings in TimesNet. Is this an oversight in the comparative experiment?

Besides, I am very interested in your review comments on this work and the article. Could you please have an in-depth exchange?
Looking forward to your reply。

**Comment by stitsyuk** [comment]
Dear cooker-coder,
First of all, thank you for reading our paper carefully and for raising this important point.

Regarding the unified protocol settings in TimesNet, we would like to clarify that its unified hyperparameter experimental setup includes parameters such as the historical window and prediction length. However, parameters like batch size, learning rate, and the number of training epochs are subject to the available GPU memory and computational resources. This is because these parameters primarily affect the efficiency and speed of training rather than the nature of the input data itself. In our understanding, the term "unified hyperparameter setting" primarily refers to ensuring consistent input data conditions across all models — a condition that our experiments fully satisfy.

If you refer to the official TimesNet implementation, you will notice that although it is a unified hyperparameter setting, such parameters as batch size, learning rate, and epochs are adapted depending on the model and dataset. For example (b - batch size, lr - learning rate, e - number of training epochs):
* TimeMixer ETT: b = 16, lr = 0.01
* TimeXer ETT: b = 4
* WPMixer ETT: b = 256, lr = 0.000242438, e = 60
* TimeMixer weather: b = 128, lr = 0.01
* WPMixer weather: b = [32, 64, 32, 128], lr = [0.000913333, 0.001379042, 0.000607991, 0.001470479], e = 60
* and many more

As for review comments, could I kindly ask you to directly contact the first author (stitsyuk@kaist.ac.kr) regarding these details? Since AAAI does not publicly disclose reviews, we believe such discussions are better handled over email.

Thank you again for your thoughtful reading and interest in our work. I hope this clarifies your concern!

Best regards,
Artyom Stitsyuk

**Comment by cooker-coder** [comment]
Thank you very much for your timely reply.
First of all, I agree with your point :" **_In our understanding, the term "unified hyperparameter setting" primarily refers to ensuring consistent input data conditions across all  "Models-a condition that our experiments fully satisfy._**" In my opinion, for the unified experimental setup experiment, the retrospective window and the predicted length should be aligned, and the rest should be adjusted as hyperparameters.
Secondly, I think the batch size and training rounds will affect the experimental results of the model to varying degrees. (At least on my model)
In addition, in the example you gave, WPMixer is actually the parameter setting in the author's hyperparameter search experiment.

![Image](https://github.com/user-attachments/assets/a0969422-8c35-4f62-8797-a5773bc00698)

In conclusion, I think the problem of unified experimental setup is caused by the fact that a corresponding unified consensus has not yet been formed in this field. But your answer answered my doubts very well.
Finally, thank you very much for your reply. I also hope to learn some relevant review opinions about this work. I will contact you via email.
